app/models/foreman_tasks/triggering.rb

```diff
@@ -39,11 +40,11 @@ class Triggering < ActiveRecord::Base
     validate :correct_cronline, :if => Proc.new { |t| t.recurring? && t.input_type == :cronline }
 
     def self.new_from_params(params = {})
-      self.new(params).tap do |targeting|
+      self.new(params.except(:mode)).tap do |targeting|
```

Comment:
When trying to rerun job I'm getting `Can't mass-assign protected attributes: start_at, start_before`
`params.except(:mode, :start_at, :start_before)` fixes the issue